plugins/auth-backend/src/lib/oauth/OAuthAdapter.ts

```diff
@@ -155,7 +155,7 @@ export class OAuthAdapter implements AuthProviderRouteHandlers {
       // remove refresh token cookie before logout
       this.removeRefreshTokenCookie(res);
     }
-    res.send('logout!');
+    res.status(204).json('logout!');
```

Comment:
not json

Reply:
Also note that the status 204 doesn’t allow a message body (https://tools.ietf.org/html/rfc7231#section-6.3.5). Should better be 200 or should skip the „logout!“